### DISCUSSION TITLE
Actividad 0 - Zelayeta Iñaki José

### DISCUSSION THREAD

**Discussion by IZelayeta** [frlp-utn-ingsoft/catedra#49] [Show and tell]
https://github.com/IZelayeta/Ingenieria-y-Calidad-de-software.git

**Comment by RodrigoJacznik** [comment]
Hola,
los commits podrían seguir mejor el formato (todos te quedaron en parte 1) pero son bastante atómicos y descriptivos.
La app funciona bien.
Actividad aprobada.

**Comment by IZelayeta** [comment]
Hola profesor, buen día, muchas gracias por la corrección, lo tendré en cuenta para las siguientes actividades. 
    El lunes, 15 de abril de 2024, 00:00:21 ART, Rodrigo Jacznik ***@***.***> escribió:  
 
 


Hola,
los commits podrían seguir mejor el formato (todos te quedaron en parte 1) pero son bastante atómicos y descriptivos.
La app funciona bien.
Actividad aprobada.
